cmd/odfcli/set/recovery-profile.go

```diff
@@ -0,0 +1,47 @@
+package set
+
+import (
+	"fmt"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odfcli"
+	"github.com/red-hat-storage/odf-cli/pkg/set"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+var validRecoveryOptions = []string{"high_client_ops", "high_recovery_ops", "balanced"}
+
+var recoveryProfile = &cobra.Command{
+	Use:                "recovery-profile",
+	Short:              "Set the recovery profile to favor new IO, recovery, or balanced mode with options high_client_ops, high_recovery_ops, or balanced",
+	DisableFlagParsing: true,
+	Args:               cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		var recoveryInput string
+		if len(args) == 0 {
+			recoveryInput = "balanced"
+		} else {
+			recoveryInput = args[0]
+		}
```

Comment:
Related to my other comment. I think the behavior of re-setting the default when no value is given is one that could cause a problem for a user who intends to set a value but forgets to include it. 

Consider this scenario:
- I am a support engineer working with a customer who needs to speed up Ceph recovery ops
- I advise the customer to use `odf-cli set recover-profile` to set high recovery ops
- The customer misinterprets this, and only runs `odf-cli set recover-profile`
- The command executes successfully, leading the customer to believe they have done the right thing
- The customer tells the support engineer they did it
- The customer's issue still persists, and the support engineer, believing the customer did the right thing, proceeds to debug other things that are not the root cause

If, instead, we make it so the value selection is mandatory, the customer cannot accidentally apply the default unexpectedly, and we could save customers, support engineers, or ourselves a lot of time in the future.

@travisn thoughts?

Reply:
Sounds good to keep it required, agreed it's better this way to avoid unintended settings.

Reply:
done.